raiwidgets/raiwidgets/dashboard.py

```diff
@@ -74,7 +75,8 @@ def __init__(self, *,
             'id': self.id,
             'baseUrl': self._service.env.base_url,
             'withCredentials': self._service.with_credentials,
-            'locale': locale
+            'locale': locale,
+            'featureFlights': kwargs.get('feature_flights')
```

Comment:
do you need to validate the type of the parameter perhaps before passing it to the UX?

Reply:
especially since this is set by the user which can be error-prone

Reply:
wait, is it a string, or a list of strings?  I thought it was a list of strings?  And we can also validate the strings are expected perhaps?

Reply:
how does this work with multiple feature flights?  can you pass multiple flights in one parameter?

Reply:
oh I see, you do a split on the separator, which is an ampersand:
```
return featureFlights.split(featureFlightSeparator)
```
might it not be easier to specify all flights as a list of strings in the python api?  Or is this getting too particular about the user interface which we don't expect users to use anyway?

Reply:
I guess you can add a test to check that it works with a string, and add a validation method to check that it indeed is a string (and not, say, a list) and that it doesn't have weird spaces/characters etc, other than ampersands?

Reply:
Correct, it's not expected to be used by anyone other than us. I initially wanted it to be a list of strings to be easy to use. However, it's easiest to use IAppSetting in the test env and that is just strings. Since we pass flights as a string in Studio anyway I figured this isn't the worst solution. If someone gets it wrong that's really on them (or whoever gave them that flight string). I'll write the contributor guide section as you suggested, that should be sufficient. I'll validate it's a string as well.